README.md

```diff
@@ -19,6 +19,19 @@ The projects the team are working on in the January 2018 firebreak are:
 **deployed prototype:** [https://registers-upvote.herokuapp.com/registers](https://registers-upvote.herokuapp.com/registers)
 
 
+## Registers Sprint 
+**A sprint-by-sprint list of what we're doing and learning that:**
+- documents achieved milestones for each sprint
+- documents user research findings
+
+**code:** [https://github.com/openregister/firebrakeproject](https://github.com/openregister/firebrakeproject)
```

Comment:
can you rename the repo, the name is kind of meaningless, can you call it `sprint-notes` or something?

Reply:
Thanks @gidsg, I have changed the name of the Repo